Leibit.Core/Data/sued/TF.xml

```diff
@@ -0,0 +1,412 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<estw>
+  <station name="Radolfzell" short="RRZ" refNr="56">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="66BP" direction="right" />
+      <block name="56BN2" direction="right" />
+    </track>
+  </station>
+
+  <station name="Schwackenreute" short="RSU" refNr="65">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="65BN" direction="left"/>
+    </track>
+  </station>
+
+  <station name="Eriskirch" short="TEK" refNr="76">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="77BP2" direction="left"/>
```

Comment:
Dieser Block ist hier doppelt zugeordnet, denn er ist unten auch Friedrichshafen Ost zugeordnet. Das erste Zugnummernfeld, dass die Züge aus Richtung Eriskirch erreichen, müsste "1" sein (die genaue Bezeichnung habe ich gerade nicht zur Hand, wahrscheinlich 77B1 oder 76B1). Dieses Feld sollte dann auch hier angegeben werden, während 77BP2 zu Friedrichshafen Ost gehört.

Reply:
tatsächlich ist das wohl nur mit "1" bezeichnet. Ich hab das mal angepasst.
![grafik](https://user-images.githubusercontent.com/7548596/91752602-deaaaa00-ebc6-11ea-9696-e171cd880e7e.png)

Reply:
Ausschlaggebend ist nicht die Bezeichnung in der Sim, sondern in der Datei "LEIBIT_FRIEDRICHSHAFEN.dat". Bei Spiegelfeldern stimmen die nicht überein. Ich tippe hier mal auf "76S1" oder "77S1".

Reply:
Bin grad mal in der Demo gucken gegangen. Dort heißt es "77S1".

Reply:
Meinst du damit die Datei, die immer mal wieder im ESTWonline Ordner auftaucht? die liegt dort bei mir nur für den Bruchteil einer Sekunde, habe also keine Chance die Anzusehen. Gibts da einen Trick?

Reply:
Ja, genau die meine ich. Am besten du beendest mal beide ESTWonline's (oder klickst auf "Anhalten") gehst dann im Ordner "Kommunikation" des ESTWs gucken. Zum Test am besten mal eine ZN in dem fraglichen Feld eintippen und dann gucken, mit welchem Blocknamen sie in der Datei auftaucht. Ggf. muss die Datei einmal von Hand gelöscht werden, da ESTWsim die Datei nicht automatisch überschreibt.

Reply:
Super, das hat funktioniert! Ich prüf dann nochmal alle IDs.

Reply:
Sehr gut. Bei allen Nicht-Spiegelfeldern stimmt das aber zu 99% mit der Bezeichnung in der Sim überein 😉